gravitee-apim-portal-webui-next/src/styles.scss

```diff
@@ -37,4 +37,7 @@ body {
   background-color: $background-color;
   color: $default-text-color;
   font-family: 'Golos UI', Roboto, 'Helvetica Neue', sans-serif;
+
+  // mat-card
+  --mdc-elevated-card-container-elevation: 0;
```

Comment:
Later on we can move that to the seperate file to keep it clean, I created some basic implementation for this so there is not point to do it here